pkg/server/api/logger/v1/service.go

```diff
@@ -0,0 +1,60 @@
+package logger
+
+import (
+	"context"
+
+	loggerv1 "github.com/spiffe/spire-api-sdk/proto/spire/api/server/logger/v1"
+	"github.com/spiffe/spire-api-sdk/proto/spire/api/types"
+
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+)
+
+type Config struct {
```

Comment:
The implementation is currently touching the "std" logger provided by the logrus package through exported package-level methods. The `std` logger is primarily intended to interface with the go standard library `log` package. In any case, it isn't the logger that is configured for the server in run.go.

I'd suggest plumbing the *logrus.Logger set up in run.go all the way into this package and invoking methods on that to influence its LogLevel.

Reply:
Testing seems to indicate that the std logger you are referring to is being used, as the logs are directly impacted when calls to the build executables suppress lower log messages or are tuned to expose them.

However, it reads better to pass the logrus.Logger through the api layers, and I'm happy to make the changes.  That said, I highly suspect that the logger that is configured for the server in run.go either is the std logger reconfigured or chains into to the std logger; otherwise, my testing would not have been able to suppress and then expose different logs at different log levels.

Or, my testing might be wrong.  I might be testing a different set of log output than the one you're considering.  If that's so, please reach out to me via chat so we can understand each other on this matter.